sycl/source/detail/device_impl.hpp

```diff
@@ -242,6 +242,10 @@ class device_impl {
   /// @throw sycl::feature_not_supported if feature is not supported on device
   uint64_t getCurrentDeviceTime();
 
+  /// Check clGetDeviceAndHostTimer is available for fallback profiling
+
```

Comment:
@eunakim0103 we discussed with you offline a couple of days ago how it should be properly addressed, still, didn't fixed.